cmd/derper/cert.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
```

Comment:
wrong year

Reply:
(only set it to the current year for new files, not on existing files)

Reply:
fixed